examples/http-request-latency-burnrate.jsonnet

```diff
@@ -0,0 +1,18 @@
+local latency = import '../slo-libsonnet/latency-burn.libsonnet';
+
+{
+  local query = latency.burn({
+    metric: 'http_request_duration_seconds_bucket',
+    selectors: ['job="telemeter-server"','handler="upload"'],
```

Comment:
Before merging these selectors need to be more like an example :)
See the selectors from examples/http-request-error-burnrate.jsonnet

Reply:
yep , updated and this simplified the whole thing quite a bit.